src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    abstract void draw();
```

Comment:
![Uploading Screenshot 2022-07-18 at 08.43.54.png…]()

Reply:
[Interface Drawer created](https://github.com/Maksym-Kulakov/jv-oop-advanced/commit/1a4fe8a9a34088543ecaac305af01776afaf60b7)

Reply:
Unfortunately I can`t open yr .png file with changes rqrd. But intuitively I have considered that draw() meth to be transphered fm Parent class to separate Interface. I hope it`s correct way.